notes-service/src/main/java/org/exoplatform/wiki/jpa/JPADataStorage.java

```diff
@@ -287,6 +288,19 @@ public Page getParentPageOf(Page page) throws WikiException {
         childrenPages.add(convertPageEntityToPage(childPageEntity));
       }
     }
+    boolean isRootPage = pageEntity.getParentPage() == null;
+    List<DraftPageEntity> draftPageEntities;
+    if (isRootPage) {
+      draftPageEntities = draftPageDAO.findRootDraftPagesByUser(ConversationState.getCurrent().getIdentity().getUserId());
```

Comment:
This should not be used in Storage layer

Reply:
It seems important architecturally. Do we have an item for this in the dev guidelines ?
CC @boubaker

Reply:
I'am fixing this in the PR to be merged to develop, sorry i didn't see the comment. cc @ahamdi